libraries/node-core-library/src/LockFile.ts

```diff
@@ -85,6 +85,28 @@ export class LockFile {
     throw new Error(`File locking not implemented for platform: "${process.platform}"`);
   }
 
+  /**
+   * Attempts to create the lockfile.
+   * Will continue to loop at the given interval until the lock becomes available.
+   * Note that this function is subject to starvation issues.
```

Comment:
>* Note that t [](start = 3, length = 13)

Use \@remarks and be a little more specific what people should do with that knowledge